action/envelope.go

```diff
@@ -25,6 +26,7 @@ type (
 		LoadProto(pbAct *iotextypes.ActionCore) error
 		SetNonce(n uint64)
 		SetChainID(chainID uint32)
+		ToRLP() (RlpTransaction, error)
```

Comment:
will this be used outside of action package? if not, use `toRLP()`

Reply:
https://github.com/iotexproject/iotex-core/pull/3070/files#diff-1ee1383260f91d9a4082df0e44f3e1be95e37a8930d01712b65f7e8756b1d0e9R217